packages/components/src/Tile/TileGrid/_docs/TileGrid.stories.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react"
+import { Meta, StoryObj } from "@storybook/react"
+import { InformationTile } from "~components/Tile"
+import { TileGrid } from "../index"
+
+const meta = {
+  title: "Components/Tiles/TileGrid",
```

Comment:
We have some inconsistencies in Storybook, are we doing spaces or no spaces with component names here?

Reply:
Yeah, midway we decided to ditch spaces lol. Too much effort to adhere to some request from eons ago 😄